BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  File.swift
+//  
+//
+//  Created by Winnie Teichmann on 2024-03-22.
+//
+
+import Common
+import UIKit
+
+class ToolbarButton: UIButton, ThemeApplicable {
+    public struct UX {
+        public static let verticalInset: CGFloat = 8
+        public static let horizontalInset: CGFloat = 8
+    }
+
+    var foregroundColorNormal: UIColor = .clear
+    var foregroundColorHighlighted: UIColor = .clear
+    var foregroundColorDisabled: UIColor = .clear
+    var backgroundColorNormal: UIColor = .clear
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        configuration = UIButton.Configuration.plain()
+        configuration?.contentInsets = NSDirectionalEdgeInsets(top: UX.verticalInset,
+                                                               leading: UX.horizontalInset,
+                                                               bottom: UX.verticalInset,
+                                                               trailing: UX.horizontalInset)
+    }
+
+    open func configure(element: ToolbarElement) {
+        guard let config = configuration else {
+            return
+        }
+        var updatedConfiguration = config
+
+        let image = UIImage(named: element.iconName)?.withRenderingMode(.alwaysTemplate)
+        let action = UIAction(title: element.a11yLabel,
+                              image: image,
+                              handler: { _ in
+            guard let closure = element.onSelected else { return }
+            closure()
```

Comment:
A suggestion, not a big deal, but if there are no other operations in the handler that depend on `onSelected` being non-nil, why not just calling `element.onSelected?()`. This change could make the code more concise.